source/Nevermore/UpgradeScripts/Script0003-Nevermore-RelatedDocumentType.sql

```diff
@@ -0,0 +1,2 @@
+﻿IF NOT EXISTS (SELECT NULL FROM sys.table_types WHERE name = 'RelatedDocumentTableValuedParameter')

```

Comment:
Made the upgrade scripts safe to re-run and added `Nevermore` to the name, because we're going to start including them inside Server and Octofront